orientation/faq.md

```diff
@@ -0,0 +1,215 @@
+
+</details>
+
+<details>
+  <summary>Does it pay to become a marketplace operator?</summary>
+
+Yes. Marketplace operators benefit from earning commission on marketplace transactions related to data consumption as well as staking. Ocean Market is primarily focussed on monetising data however it is also designed to handle the sale of any digital asset or service. As a result the total addressable market goes way beyond revenues from just selling data. Operating costs for an Ocean-powered marketplace are moderate and the base code is open source and available free of charge under the Apache 2 license.
+
+</details>
+
+<details>
+  <summary>Why Publish?</summary>
+
+Publishing data, algorithms and other digital assets and services on an Ocean-powered marketplace offers numerous opportunities to earn on the future revenue streams connected to that data as well as build lucrative ecosystem that add value to the published asset. It also allows for the discovery and insights into new use cases and applications of the published asset.
+
+</details>
+
+<details>
+  <summary>What about the price fluctuation of Ocean?</summary>
+
+Price fluctuation is mitigated through the use of the Ocean backed stable coin H2O.
+
+</details>
+
+<details>
+  <summary>Who pays for gas fees?</summary>
+
+Gas fees for marketplace transactions are paid by the user initiating the transaction (for publishing, consuming, staking, etc).
+
+</details>
+
+<details>
+  <summary>Where do the docker containers run?</summary>
+
+Dockers containers can run anywhere. Ocean Market use a docker run by the Ocean Protocol Foundation OPF); limit: 1 CPU limit / 60 seconds max. NOTE: This means OPF technically has access to data. In the case of a forked Ocean-powered marketplace the owner of marketplace must set up computation environment. If individual users of the marketplace are concerned with security they should be prepared to host both the data and provide compute-to-data services on premise.
+
+</details>
+
+<details>
+  <summary>Who pays for the computation?</summary>
+
+The marketplace owner.
+
+</details>
+
+<details>
+  <summary>What cryptocurrency do I need for transactions?</summary>
+
+The type if cryptocurrencies needed for transactions on the marketplace depends on which network(s) the marketplace is running (Ethereum, Polygon, EWT, BSC, Moonriver, etc.). Regardless of network, users will need to have Ocean tokens as well as the corresponding network token, which is used to pay for gas.
+
+</details>
+
+<details>
+  <summary>Can I use the off the shelf CSS available in the repo?</summary>
+
+Marketplace name, logo and typeface must be changed by the client. Slight modification would be enough for compliance. For more information consult the READ ME file on GitHub. https://github.com/oceanprotocol/market#-forking
+
+</details>
+
+<details>
+  <summary>Roadmap - what’s to come in 2022?</summary>
+
+- Q4 will decouple price from the liquidity pool.
```

Comment:
add veOCEAN

Reply:
I've changed this to a more generic answer that points people to the roadmap page on the main site. That way we don't need to keep updating the roadmap in multiple places